src/routes/sessions/[id]/Controls.svelte

```diff
@@ -43,52 +36,9 @@
 	// HACK: Stop is a `string[]` so we are hardcoding it to a single value for now
 	let stop: string = $session.options.stop?.[0] ?? '';
 	$: if (stop) $session.options.stop = [stop];
-
-	let knowledgeId: string | undefined;
-
-	$: {
-		if ($session.systemPrompt.knowledge && !knowledgeId) {
-			// Initial load: set knowledgeId if knowledge exists
-			knowledgeId = $session.systemPrompt.knowledge.id;
-		} else if (knowledgeId !== $session.systemPrompt.knowledge?.id) {
-			// Knowledge selection changed
-			if (knowledgeId) {
-				const knowledge = loadKnowledge(knowledgeId);
-				$session.systemPrompt.knowledge = knowledge;
-				$session.systemPrompt.content = knowledge.content;
-			} else {
-				// Clear knowledge if knowledgeId is undefined
-				$session.systemPrompt.knowledge = undefined;
-				$session.systemPrompt.content = '';
-			}
-		}
-	}
 </script>
 
 <div class="controls">
-	<Fieldset>
-		<P><strong>{$LL.systemPrompt()}</strong></P>
-		<FieldSelect
-			label={$LL.knowledge()}
-			name="knowledge"
-			disabled={!$knowledgeStore.length}
-			placeholder={!$knowledgeStore.length ? $LL.emptyKnowledge() : ''}
-			options={$knowledgeStore?.map((k) => ({ value: k.id, label: k.name }))}
-			bind:value={knowledgeId}
-		>
-			<svelte:fragment slot="nav">
-				<Button
-					aria-label={$LL.newKnowledge()}
-					variant="outline"
-					href={generateNewUrl(Sitemap.KNOWLEDGE)}
-					class="h-full text-muted"
-				>
-					<Brain class="base-icon" />
-				</Button>
-			</svelte:fragment>
-		</FieldSelect>
-	</Fieldset>
```

Comment:
We don't want to remove the system prompt, we need to restore all of this code.

Reply:
You're right, I removed by mistake. I've just added it again.